behave_django/management/commands/behave.py

```diff
@@ -179,9 +303,21 @@ def get_behave_args(self, argv=sys.argv):
 
 class BehaveArgsHelper(Command):
 
+    def __init__(self, *args, **kwargs):
+        """
+        Let's ignore any and all output that comes out of here, all the mayor
+        warning were already shown to the user inside the original Command,
+        there is no need to duplicate the messages.
+        """
+        super(BehaveArgsHelper, self).__init__(self, *args, **kwargs)
+        f = open(os.devnull, 'w')
+        self.stdout = OutputWrapper(f)
+        self.stderr = OutputWrapper(f)
```

Comment:
Wow! That _is_ a cool move ("... unless explicitly silenced"). But I'm not sure if this isn't dangerous, making people wander around desperately looking for the cause of an issue, in a weird edge case.

Can you show which kind of duplicate output we're swallowing?

Reply:
The `self.stderr` https://github.com/behave/behave-django/blob/cd8bad3d9455dcc535d3ece225dd949093d45b44/behave_django/management/commands/behave.py#L175-L187 would show up twice, but now i think because of the `warn()` that would show up twice... I'll test it later.

Reply:
Confirmed, it does not.